chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -405,6 +405,13 @@ pane.item.attachments.fileNotFound.text2.stored = It may have been moved or dele
 pane.item.attachments.fileNotFound.text2.stored.notOnServer = It may have been moved or deleted outside of %1$S, or, if the file was added on another computer, it may not yet have been synced to %2$S.
 pane.item.attachments.fileNotFound.text2.linked = It may have been moved or deleted outside of %1$S, or a Linked Attachment Base Directory may be set incorrectly on one of your computers.
 pane.item.attachments.delete.confirm  = Are you sure you want to delete this attachment?
+pane.item.attachments.autoRelink.title = Located Automatically
+pane.item.attachments.autoRelink.text1 = A matching file was found in your Linked Attachment Base Directory at the following path:
```

Comment:
Probably should show the absolute path too? As written, we're not really explaining why we're showing this dialog. It doesn't particularly matter if you just want to fix it, but it'd be clearer if we showed 1) the current absolute path and 2) the location within the LABD where it was found. Maybe `The file could not be found at the specified location, but a matching file was found in your Linked Attachment Base Directory:` and add lines for "Old Location:" and "New Location:"?

Reply:
> `The file could not be found at the specified location, but a matching file was found in your Linked Attachment Base Directory:`

Maybe with an "although"? The dialog that shows before this one already tells the user that `The attached file could not be found at the following path:` - this dialog only shows after they click `Locate...`. If this dialog _again_ says that the file couldn't be found, with slightly different wording, it sort of seems like the "Locate" operation failed, or something like that, when actually it hasn't happened yet.

Reply:
Oh, I didn't realize that this appears after `Locate…`. (I haven't actually tried running this yet.)

What about just running this check automatically when a linked file can't be found and the absolute path isn't within the current LABD?

Reply:
Sure, that makes sense.

Reply:
The only thing that worries me slightly is that finding other relinkable files in the same directory could be very slow on something like a network share. I thought giving the user a chance to back out after opening a missing file but before locating it made sense in case they decide they don't want to deal with it at the moment.

We could also fix that by breaking after one additional relinkable file is found and only performing the full search if they actually choose "Relink All." We wouldn't be able to show an exact count of other relinkable files anymore.

What do you think?

Reply:
1) We'd only be looking for other files if the current file matched.

2) We'd only check files using the same relative path segment as was required for the matched file, so whereas the first file might require multiple `stat`s, there'd only be one per file for the additional files. (I didn't look closely to see that this is what the code currently does, but I assume we can do this?)

Given these two things, I think it's OK to run automatically.

If we're concerned, we could have a time limit for checking other files and have an alternative version of the dialog that appears if we can't generate a count, but I doubt that's worth it or would even ever be shown

Reply:
Yes to both. In that case, sounds good.